tx/client.go

```diff
@@ -146,9 +86,14 @@ func NewQueryParamsBuilder() *QueryParamsBuilder {
 	return &QueryParamsBuilder{params: url.Values{}}
 }
 
-func (ub *QueryParamsBuilder) OwnedBy(owner string, transient bool) *QueryParamsBuilder {
+func (ub *QueryParamsBuilder) OwnedBy(owner string) *QueryParamsBuilder {
+	ub.params.Set("owner", owner)
+	return ub
+}
+
+func (ub *QueryParamsBuilder) OwnedByWithTransient(owner string) *QueryParamsBuilder {
```

Comment:
I prefer the original function because it looks more straightforward to me. Can't easily tell what `OwnedByWithTransient` means.

Reply:
The reason it should be separated is:
- In the case, the function has 2 params `OwnedBy(owner string, transient bool)`
For common cases, the users only want to get by the `owner`, don't care about `transient`.
They will confuse the `transient` param. They don't know should they set `true` or `false`
Moreover, `transient` param is not easy to explain for the users to understand
- It will have a similar interface with other SDKs

Reply:
If users can understand the difference between `OwnedBy` and `OwnedByWithTransient`, I think they can understand `OwnedBy(owner string, transient bool)`. The critical part is to explain what `transient` means.

I personally prefer not to use `With` style naming like `GetWithAsset` and `OwnedByWithTransient`. The function names could be more self-descriptive. Developers get used to learn how to set the required params, rather than functions.

Reply:
I agree that the critical part is to explain what transient means. Probably, we need a new name which is more self-descriptive

However, I don't prefer flag parameters. They are optional params, not required ones. I think we should give the user a chance to ignore.